.storybook/preview.tsx

```diff
@@ -54,6 +54,8 @@ const preview: Preview = {
             </OdsModulesProvider>
         ),
     ],
+
+    tags: ['autodocs'],
```

Comment:
By setting this here, can we remove it from each individual story setup?

Reply:
Removed here https://github.com/aragon/ods/pull/215/commits/2d54d377255723f4e35e6055c46210cc7261969f